src/utils.jl

```diff
@@ -57,9 +57,11 @@ function enumstr(enumname::T, t::Int32) where {T <: NamedTuple}
 end
 
 import protoc_jll
-function protoc(args=``; protoc=protoc_jll.protoc())
-    ENV′ = copy(ENV)
-    ENV′["PATH"] = string(joinpath(@__DIR__, "..", "plugin"), ":", ENV′["PATH"])
-    ENV′["JULIA"] = joinpath(Sys.BINDIR, Base.julia_exename())
-    setenv(`$protoc $args`, ENV′)
+function protoc(args=``)
```

Comment:
I thought users might want the option to pick a different protoc, but I guess the code is simple enough that they can just run it themselves if need be.

Reply:
Yes... I also changed `protoc()` to `run` the command so that it is executed in the correct environment as setup by `protoc_jll.protoc()`